Core/clim-core/frames/define-application-frame.lisp

```diff
@@ -91,11 +91,11 @@
                  (:pane                  * :conflicts (:panes :layouts))
                  (:panes                 * :conflicts (:pane))
                  (:layouts               * :conflicts (:pane))
-                 (:command-table         1)
-                 (:command-definer       1)
-                 (:menu-bar              ensure-1)
+                 (:command-table         1 :type (cons symbol list))
+                 (:command-definer       1 :type symbol)
+                 (:menu-bar              ensure-1 :type (or symbol list))
                  (:disabled-commands     *)
-                 (:top-level             1)
+                 (:top-level             1 :type (cons (or symbol cons) list))
```

Comment:
> :top-level form, where form is a list whose first element is the name of a function to be called to execute the top-level loop. The function must take at least one argument, the frame. The rest of the list consists of additional arguments to be passed to the function. The default function is default-frame-top-level.


why is there (or symbol cons)?

Reply:
The spec says

> `:top-level` *form,* where *form* is a list whose first element is **the name of a function** to be called to execute the top-level loop.

To acommodate extended function names like `(setf NAME)`, `cons` must be allowed.